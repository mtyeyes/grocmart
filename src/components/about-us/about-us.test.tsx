import React from 'react';
import { shallow } from 'enzyme';
import AboutUs from './about-us';

describe('AboutUs component', () => {
  it('should match snapshot', () => {
    const component = shallow(<AboutUs />);

    expect(component).toMatchSnapshot();
  });
});
