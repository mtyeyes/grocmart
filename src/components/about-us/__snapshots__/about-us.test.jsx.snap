// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutUs component should match snapshot 1`] = `
<section
  className="about-us"
>
  <img
    alt="Juicy fruits and crunchy vegetables"
    className="about-us__image"
    loading="lazy"
    src="test-file-stub"
  />
  <div
    className="about-us__container"
  >
    <ScrollAnimation
      animateIn="fade-in-from-right"
      animateOnce={true}
      animatePreScroll={true}
      delay={0}
      duration={1}
      initiallyVisible={false}
      offset={150}
    >
      <p
        className="about-us__hand-written"
      >
        A few words about our store
      </p>
      <h2
        className="about-us__title"
      >
        About us
      </h2>
      <p
        className="about-us__paragraph"
      >
        Grocmart is a family-owned grocery store that has been offering quality products for your everyday life since 1999, while also providing superior service and competitive prices.
      </p>
      <div
        className="about-us__bottom-wrapper"
      >
        <p
          className="about-us__name"
        >
          Sam Williams
        </p>
        <p
          className="about-us__position"
        >
          CEO, Founder
        </p>
        <img
          alt="Signature"
          className="about-us__signature"
          loading="lazy"
          src="test-file-stub"
        />
      </div>
    </ScrollAnimation>
  </div>
</section>
`;
